src/main/io/gps.c

```diff
@@ -1805,22 +2352,19 @@ static bool gpsNewFrameUBLOX(uint8_t data)
             _step++;
             _ck_b += (_ck_a += data);       // checksum byte
             _payload_length += (uint16_t)(data << 8);
-#if DEBUG_UBLOX_FRAMES
-    debug[3] = _payload_length;
-#endif
-            if (_payload_length > UBLOX_PAYLOAD_SIZE) {
-                _skip_packet = true;
-            }
+
             _payload_counter = 0;   // prepare to receive payload
             if (_payload_length == 0) {
                 _step = 7;
             }
+            _buffer.bytes[_payload_counter] = data;
```

Comment:
data is second byte of payload length, no reason to store it

Reply:
have reverted the recent change, is this still needed?